jccli/cli/system.py

```diff
@@ -0,0 +1,81 @@
+import json
+import click
+from jccli.jc_api_v1 import JumpcloudApiV1
+
+
+@click.group()
+@click.pass_context
+def system(ctx):
+    """
+    Command set for systems
+    """
+    pass
+
+
+@system.command("get")
+@click.option('--id', '-i', 'system_id', required=True, type=str)
+@click.pass_context
+def get_system(ctx, system_id):
+    """
+    Detail view of system, outputted in JSON.
+    """
+    api1 = JumpcloudApiV1(ctx.obj.get('key'))
+    response = api1.get_system(system_id=system_id)
+    serialized_response = json.dumps(response, indent=2)
+    click.echo(f"{serialized_response}")
+
+
+@system.command('list')
+@click.option('--active/--inactive', type=bool, default=None)
+@click.option('--arch', type=str, default=None, help='architecture')
+@click.option('--display-name', 'displayName', type=str, default=None)
+@click.option('--hostname', type=str, default=None)
+@click.option('--os', type=str, default=None)
+@click.option('--remote-ip', 'remoteIp', default=None)
+@click.pass_context
+def list_systems(ctx, **kwargs):
+    """
+    List view of systems, outputted in JSON
+    """
+    filter = {}
```

Comment:
we need to provide some documentation to the user on creating a valid filter.   Any ideas on how we can do that?

Also my assumption of a `list` command is to return all systems when command is executed without passing any parameters.  However it looks like the default is turn no systems if not filter is passed in, correct?  This applies to `list_users` now as well.

Reply:
Added some info to docstring of `jccli system list`. Tell me what you think

Reply:
looks good.